CHANGELOG.md

```diff
@@ -417,156 +449,156 @@ rocRAND 2.10.17 for ROCm 6.0.0
-- Improved documentation
-- Improved verbose output during argument checking on API function calls
+* Function arguments for `rocsparse_spmv`
+* Function arguments for `rocsparse_xbsrmv` routines
+* When using host pointer mode, you must now call `hipStreamSynchronize` following `doti`, `dotci`,
+  `spvv`, and `csr2ell`
+* Improved documentation
+* Improved verbose output during argument checking on API function calls
 
-##### Deprecated
+##### Removals
 
-- Deprecated rocsparse_spmv_ex
-- Deprecated rocsparse_xbsrmv_ex routines
+* Auto stages from `spmv`, `spmm`, `spgemm`, `spsv`, `spsm`, and `spitsv`
+* Formerly deprecated `rocsparse_spmm_ex` routine
 
-##### Removed
+### Fixes
 
-- Removed auto stages from spmv, spmm, spgemm, spsv, spsm, and spitsv.
-- Removed rocsparse_spmm_ex routine
+* Bug in `rocsparse-bench` where the SpMV algorithm was not taken into account in CSR format
+* BSR and GEBSR routines (`bsrmv`, `bsrsv`, `bsrmm`, `bsrgeam`, `gebsrmv`, `gebsrmm`) didn't always
+  show `block_dim==0` as an invalid size
+* Passing `nnz = 0` to `doti` or `dotci` wasn't always returning a dot product of 0
 
-##### Fixed
+### Additions
 
-- Fixed a bug in rocsparse-bench, where SpMV algorithm was not taken into account in CSR format
-- Fixed the BSR/GEBSR routines bsrmv, bsrsv, bsrmm, bsrgeam, gebsrmv, gebsrmm so that block_dim==0 is considered an invalid size
-- Fixed bug where passing nnz = 0 to doti or dotci did not always return a dot product of 0
+* `rocsparse_inverse_permutation`
+* Mixed-precisions for SpVV
+* Uniform int8 precision for gather and scatter
 
 #### rocThrust 3.0.0
 
 rocThrust 3.0.0 for ROCm 6.0.0
 
-##### Added
-
-- Updated to match upstream Thrust 2.0.1
-- NV_IF_TARGET macro from libcu++ for NVIDIA backend and HIP implementation for HIP backend.
-
-##### Changed
+##### Additions
 
-- The cmake build system now additionally accepts `GPU_TARGETS` in addition to `AMDGPU_TARGETS` for
-  setting the targeted gpu architectures. `GPU_TARGETS=all` will compile for all supported architectures.
-  `AMDGPU_TARGETS` is only provided for backwards compatibility, `GPU_TARGETS` should be preferred.
+* Updated to match upstream Thrust 2.0.1
+* `NV_IF_TARGET` macro from libcu++ for NVIDIA backend and HIP implementation for HIP backend
 
-##### Removed
+##### Changes
 
-- Removed cub symlink from the root of the repository.
-- Removed support for deprecated macros (THRUST_DEVICE_BACKEND and THRUST_HOST_BACKEND).
+* The CMake build system now accepts `GPU_TARGETS` in addition to `AMDGPU_TARGETS` for
+  setting targeted GPU architectures
+  * `GPU_TARGETS=all` compiles for all supported architectures
+  * `AMDGPU_TARGETS` is only provided for backwards compatibility (`GPU_TARGETS` is preferred)
+* Removed CUB symlink from the root of the repository
+* Removed support for deprecated macros (`THRUST_DEVICE_BACKEND` and
+  `THRUST_HOST_BACKEND`)
 
-##### Known Issues
+##### Known issues
```

Comment:
Do we want to call this section Deprecations? Not sure if this is a known issue.

@LisaDelaney